components/switch/style/index.less

```diff
@@ -107,7 +107,7 @@
     &:before,
     &:after {
       left: 100%;
-      margin-left: -12.5px;
+      margin-left: -(@switch-sm-height - 3px);
```

Comment:
Sorry to have a little doubt: Why not define a new variable but use the `switch-sm-height`(Inconsistent with single responsibility)?
ps: `.5px` may be related to browser compatibility. (cc @afc163 )

Reply:
Ah, sure thing. I noticed currently that variable was being used with multiple responsibility.